src/cli/init.rs

```diff
@@ -111,7 +166,46 @@ pub async fn execute(args: Args) -> miette::Result<()> {
             },
         )
         .unwrap();
-    fs::write(&manifest_path, rv).into_diagnostic()?;
+
+    if conda_deps.is_empty() && pypi_deps.is_empty() {
+        fs::write(&manifest_path, rv).into_diagnostic()?;
+    } else {
```

Comment:
This seems like an optimization that is not really needed (especially since you potentially load the project anyways below). Just remove the if else and always do the code in the else.